datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -3041,4 +3048,96 @@ mod tests {
         run_test_with_spill_pool_if_necessary(20_000, false).await?;
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_statistic() -> Result<()> {
```

Comment:
It's better to have the real run statistics, see here: https://github.com/apache/datafusion/blob/main/datafusion/core/tests/physical_optimizer/partition_statistics.rs#L233-L235

Reply:
I also suggest moving the test into one place like other plans: https://github.com/apache/datafusion/blob/main/datafusion/core/tests/physical_optimizer/partition_statistics.rs